Utilities/General/ibeos/das_client

```diff
@@ -64,19 +64,19 @@ if [ "${FORMAT}" = "json" ]  ; then
   else
     echo "${QUERY_RESULTS}" > ${DAS_QUERY_DIR}/${QUERY_SHA_HASH}.out
   fi
+  cat ${DAS_QUERY_DIR}/${QUERY_SHA_HASH}.out
+  return
```

Comment:
@kpedro88 , I think this will fail as we are not in function. Change this to `exit 0`